osf/migrations/0182_add_education_employment_models.py

```diff
@@ -2,100 +2,12 @@
-
-
-def parse_model_datetime(month, year):
-    try:
-        parsed = datetime.strptime('{} {}'.format(month, year), '%m %Y')
-    except ValueError:
-        parsed = datetime.strptime('{} {}'.format(month, year), '%m %y')
-    except ValueError:
-        parsed = None
-    return parsed
-
-
-def set_model_content(model, queryset, original_attribute):
-    for user in queryset:
-        original_entries = getattr(user, original_attribute)
-        for entry in original_entries:
-            institution = entry.get('institution')
-            if institution:
-                new_object = model(
-                    user=user,
-                    institution=institution
-                )
-                for key, value in entry.items():
-                    if key != 'institution' and key not in DATE_FIELDS:
-                        setattr(new_object, key, value)
-
-                start_year = entry.get('startYear', None)
-                start_month = entry['startMonth'] if start_year else None
-                if start_year and start_month:
-                    new_object.start_date = parse_model_datetime(start_month, start_year)
-
-                end_year = entry.get('endYear', None)
-                end_month = entry['endMonth'] if end_year else None
-                if end_year and end_month:
-                    new_object.end_date = parse_model_datetime(end_month, end_year)
-
-                new_object.save()
-
-def reset_field_content(model, queryset, original_attribute):
-    for entry in queryset:
-        user = entry.user
-        start_date = entry.start_date
-        end_date = entry.end_date
-        attributes = {
-            'institution': entry.institution,
-            'startYear': start_date.year if start_date else None,
-            'startMonth': start_date.month if start_date else None,
-            'endYear': end_date.year if end_date else None,
-            'endMonth': end_date.month if end_date else None,
-            'department': entry.department,
-            'ongoing': entry.ongoing,
-        }
-        if original_attribute == 'schools':
-            attributes['degree'] = entry.degree
-        elif original_attribute == 'jobs':
-            attributes['title'] = entry.title
-
-        user_entries = getattr(user, original_attribute, [])
-        user_entries.append(attributes)
-        setattr(user, original_attribute, user_entries)
-        user.save()
-
-def put_jobs_and_schools_back(state, schema):
-    UserEducation = state.get_model('osf', 'usereducation')
-    UserEmployment = state.get_model('osf', 'useremployment')
-
-    education_queryset = UserEducation.objects.all()
-    reset_field_content(UserEducation, education_queryset, 'schools')
-
-    employment_queryset = UserEmployment.objects.all()
-    reset_field_content(UserEmployment, employment_queryset, 'jobs')
-
+from osf.management.commands import populate_new_models, put_jobs_and_schools_back
```

Comment:
Could you add in a flag and a check on L147 to not run data migrations unless that flag is set? It should be off by default, but local-dist.py should have it set to true. This will allow us to say that Production won't get the migration automatically, but staging and local environments can get that migration without running the management command.